server/AccountManagement.java

```diff
@@ -1,48 +1,114 @@
 package newbank.server;
 
+import java.util.ArrayList;
+
 public class AccountManagement {
 
-    public String showMyAccounts(Customer customer) {
-        return customer.accountsToString();
+    Account selectedAccount;
+
+    public String showMyAccounts(Customer customer, NewBankClientHandler newBankClientHandler) {
+        newBankClientHandler.sendOutput("Selected: Show existing accounts and balance\n");
+        newBankClientHandler.sendOutput(customer.accountsToString());
+        newBankClientHandler.sendOutput("\nExiting to Customer Menu.");
+        return newBankClientHandler.printCustomerMenu();
     }
-    public String newAccount(Customer customer, NewBankClientHandler newBankClientHandler){
 
-        boolean valid = false;
-        String newAccountName = "";
+    public String newAccount(Customer customer, NewBankClientHandler newBankClientHandler) {
+
+        boolean valid = true;
```

Comment:
What's the logic here, why invert the `valid`? It made sense previously, that you start from an invalid state, and as you progress and do the checks, if everything went ok, it becomes valid...

Reply:
hmmm. Not sure why I changed it. It made more sense to me when I was doing it, but you are right. 
Addressed.